force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,7 +171,24 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
+        String gradeResult;
+
+        if (score >= 90){
+            gradeResult = 'A';
+        }
+        if (score >=80 && score <= 89){
```

Comment:
If it has made it past your first check, you know it is not over 90, so the second part of this to make sure  <= 89 would be unnecessary if you use if else statements here.